src/proxy-ui-api/frontend/src/views/Clients/ServiceClients/ServiceClientAccessRights.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div class="xrd-tab-max-width xrd-view-common">
+    <subViewTitle :title="serviceClientId" @close="close" />
+
+    <v-card flat>
+      <table class="xrd-table service-client-margin">
+        <thead>
+        <tr>
+          <th>{{$t('serviceClients.name')}}</th>
+          <th>{{$t('serviceClients.id')}}</th>
+        </tr>
+        </thead>
+        <tr>
+          <td>{{serviceClient.name}}</td>
+          <td>{{serviceClient.id}}</td>
+        </tr>
+      </table>
+    </v-card>
+
+
+    <div class="group-members-row">
+      <div class="row-title">{{$t('serviceClients.accessRights')}}</div>
+      <div class="row-buttons">
+        <large-button
+          @click="removeAll()"
+          outlined
+          data-test="remove-all-access-rights"
+          v-if="accessRights.length > 0"
+        >{{$t('serviceClients.removeAll')}}
+        </large-button>
+        <large-button
+          @click="addService()"
+          outlined
+          data-test="add-subjects-dialog"
+        >{{$t('serviceClients.addService')}}
+        </large-button>
+      </div>
+    </div>
+
+    <table class="xrd-table service-client-margin" v-if="accessRights.length > 0">
+      <thead>
+        <tr>
+          <th>{{$t('serviceClients.serviceCode')}}</th>
+          <th>{{$t('serviceClients.title')}}</th>
+          <th>{{$t('serviceClients.accessRightsGiven')}}</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="accessRight in accessRights">
```

Comment:
If the accessRight has an id or other unique thing it would be good to use it with v-bind:key
[vue/require-v-for-key]
Elements in iteration expect to have 'v-bind:key' directives.

Reply:
In this case there is no id to use. I added v-bind:index.